resources/styles/book-content/base.less

```diff
@@ -142,6 +138,12 @@ figure {
       width: 100%;
     }
   }
+  // splash images inside are always full width
```

Comment:
Does that make sense? I struggled with how much detail to go into there while still making the point somewhat concisely.